addons/ballistics/CfgWeapons.hpp

```diff
@@ -51,12 +51,15 @@ class CfgWeapons {
     class DMR_03_base_F: Rifle_Long_Base_F {
         ACE_barrelLength = 508.0;
         ACE_barrelTwist = 254.0;
-        initSpeed = -0.991536;
         magazineWell[] += {
             "CBA_762x51_M14"
         }; // vanilla magazineWell[] = {"M14_762x51"};
     };
 
+    class srifle_DMR_03_F: DMR_03_base_F {
+        initSpeed = -0.991536;
```

Comment:
Explain.

Reply:
When i tested the RC, i saw that the `class srifle_DMR_03_F` has his own `initSpeed `, 940 if i remember correctly.

Reply:
Yeah, saw that. I think we should keep the value in both classes and put a comment after this line.